src/widget/notificationscrollarea.cpp

```diff
@@ -66,12 +66,23 @@ void NotificationScrollArea::trackWidget(GenericChatroomWidget* widget)
     }
 }
 
-void NotificationScrollArea::updateTracking()
+/**
+ * @brief Delete notification bar to visible elements on scroll area
+ */
+void NotificationScrollArea::updateVisualTracking() {
```

Comment:
Yes. I try set default value to first argument of updateTracking, but have a truble.
But in this case I must write something like that:
```
connect(verticalScrollBar(), &QAbstractSlider::rangeChanged, this, static_cast<void (NotificationScrollArea::*)(void)>(&NotificationScrollArea::updateVisualTracking));
```